var.tfvars

```diff
@@ -113,3 +113,14 @@ use_zone_info_for_names = true # If set it to false, the zone info would not be
 #setup_snat                 = true
 
 #csi_driver_install         = false  #Set to true to enable installation of csi-driver.
+
+#luks_compliant              = false # Set it true if you prefer to use LUKS enable in OCP deployment
+#luks_config                 = [ { thumbprint = "", url = "" }, { thumbprint = "", url = "" }, { thumbprint = "", url = "" } ]
+#luks_filesystem_device      = ""  #Set the Path of device to be luks encrypted e.g. /dev/mapper/root
+#luks_format                 = ""  #Set the Format of the FileSystem to be luks encrypted  e.g. xfs
+#luks_wipeFileSystem         = ""  #Configures the FileSystem to be wiped e.g true
+#luks_device                 = ""  #Set the Path of luks encrypted partition e.g. /dev/disk/by-partlabel/root
+#luks_label                  = ""  #Set the value for user label of luks encrpted partition e.g. luks-root
+#luks_options                = []  #Set List of luks options for the luks encryption  e.g. ["--cipher", "aes-cbc-essiv:sha256"]
+#luks_wipeVolume             = ""  #Configures the luks encrypted partition to be wiped e.g. true
+#luks_name                   = ""  #Set the value for the user label of Filesystem to be luks encrypted e.g. root
```

Comment:
Instead of mentioninig the values in comments, could you please add them to the values as they are any way defaults?
Eg:
```
#luks_filesystem_device = "/dev/mapper/root" #Set the Path of device to be luks encrypted
```

Reply:
Updated code as per mentioned above.